src/Amazon.Lambda.Tools/Commands/UpdateFunctionConfigCommand.cs

```diff
@@ -401,8 +405,8 @@ private UpdateFunctionConfigurationRequest CreateConfigurationRequestIfDifferent
                 different = true;
             }
 
-            var packageType = this.GetStringValueOrDefault(this.PackageType, LambdaDefinedCommandOptions.ARGUMENT_PACKAGE_TYPE, false);
-            if (string.Equals(packageType, Lambda.PackageType.Zip.Value, StringComparison.OrdinalIgnoreCase))
+            var packageType = DeterminePackageType();
```

Comment:
Looking at the defaults file is a bug. What the code should be doing is looking at the package type via the `existingConfiguration` object.

Reply:
My initial thoughts were looking at the defaults would be required for the edge case of when a function's package type changes, but after your comment I realised that's not possible.

Updated the code to look at `existingConfiguration`